packages/universal-components/src/TextInput/TextInputTypes.js

```diff
@@ -21,9 +21,10 @@ export type Props = {|
   +minLength?: number, // this prop is supported only on web
   +onFocus?: () => void | Promise<any>,
   +onBlur?: () => void | Promise<any>,
+  +onHelpPress?: () => void,
   +onChangeText?: string => void | Promise<any>,
-  +error?: React.Node,
-  +help?: React.Node,
+  +error?: React.Node, // @TODO rename to renderError etc.?
```

Comment:
I think that just `error` is ok. It is similar to description/help or even label, prefix and suffix.

Reply:
OK

Reply:
Comment removed